README.md

```diff
@@ -81,24 +82,29 @@ These scripts can all be run from the root level of the repo:
 
-#### If using Docker:
+There are a lot of tests, so it can be helpful to constrain the tests you run locally by [using Playwright's `--grep` argument](https://playwright.dev/docs/test-cli).
 
-1. Open the Docker app, and make sure you're signed in (Docker Desktop requires a license now)
-2. Run `yarn test:browser` to begin comparing component images
-   1. If differences are detected and unexpected, evaluate your changes - we only want to update and commit references when we expect the visual changes detected
-   2. If differences are detected and expected, run `yarn test:browser:update`
-
-#### If running Locally:
+#### Updating the visual regression tests with Docker:
 
-**Note that running without using the docker image will occasionally throw false positives, depending on your system configuration. It's recommended that you utilize Docker when running DS visual regression tests.**
-
-1. If you have run `npx playwright install` and installed the playwright dependencies locally you can run the tests using their yarn commands directly.
-2. For example, to run the CMSDS VRT Tests for inteaction states: `yarn playwright test --config tests/browser/interaction.config.ts`
-3. The `-u` flag can be added to the `yarn playwright test` command to update snapshots.
+1. Open the Docker app, and make sure you're signed in (Docker Desktop requires a license now).
+2. Run `yarn test:browser` to begin comparing component images
+   1. If differences are detected and unexpected, evaluate your changes - we only want to update and commit references when we expect the visual changes detected.
+   2. If differences are detected and expected, run `yarn test:browser -u`, verify the changes, and then commit them.
+
+#### Development outside of Docker:
+
+- Run any of the browser-test sub-commands with the `--no-docker` flag, like `yarn test:browser:examples --no-docker`.
+- If you've never installed Playwright, running the `yarn test:browser` command will prompt you to install it, which you will want to do. The reason we don't need it installed when running it in Docker is because the Docker image contains all of its own dependencies.
+- Because snapshots will probably fail outside of the Docker container, pass `--ignore-snapshots` if you want to just see that the tests execute properly.
+- You can turn off headless mode by using Playwright's `--headed` flag.
+- Playwright's `--debug` flag is another helpful argument because it will pause and allow you to step through the tests.
+- Don't forget that you can run a subset of tests by using Playwright's `--grep` argument.
+- Remember that passing `--no-build` will skip re-building the source material like Storybook stories if you haven't made any changes to them and are only changing the tests themselves.
+- Here's an example of a command you might run to debug the dropdown tests: `yarn test:browser:interaction --no-docker --no-build --headed --debug --ignore-snapshots --grep "Dropdown"`
```

Comment:
I received an error running this command:
> browserType.launch: Executable doesn't exist at /Users/sarahsachs/Library/Caches/ms-playwright/chromium-1048/chrome-mac/Chromium.app/Contents/MacOS/Chromium

When running `npx playwright show-report` I'm told to run `npx playwright install` - but when I run the install command I still get the error saying I don't have the Executable